spacy/pipeline/legacy/entity_linker.py

```diff
@@ -115,7 +114,7 @@ def initialize(
         get_examples (Callable[[], Iterable[Example]]): Function that
             returns a representative sample of gold-standard Example objects.
         nlp (Language): The current nlp object the component is part of.
-        kb_loader (Callable[[Vocab], KnowledgeBase]): A function that creates a KnowledgeBase from a Vocab instance.
+        kb_loader (Callable[[Vocab], InMemoryLookupKB]): A function that creates a InMemoryLookupKB from a Vocab instance.
```

Comment:
Same question - do we need to change this type?

Reply:
Was a refactoring mistake. Changed back to `KnowledgeBase`.